bitnami/kubewatch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kubewatch
-version: 1.0.15
+version: 1.0.16
```

Comment:
Please bump minor version instead. We're usually doing so when adding new parameters to the chart.

Reply:
PTAL